app/bundles/PointBundle/Config/config.php

```diff
@@ -64,6 +64,11 @@
                 'controller' => 'MauticPointBundle:Api\PointApi:adjustPoints',
                 'method'     => 'POST',
             ],
+            'mautic_api_applyrule'    => [
+                'path'       => '/points/{id}/contact/{leadId}',
+                'controller' => 'MauticPointBundle:Api\PointApi:applyRule',
+                'method'     => 'PATCH'
```

Comment:
Why is this PATCH? It's not modifying any entity. I think that POST or maybe even GET would make more sense.